docs/book/usage/started.md

```diff
@@ -1,8 +1,277 @@
+An *epoch* is a certain period of time on the network. On Testnet, epochs are 1 day, and on
+Mainnet epochs are 2 weeks. You can extend the number of epochs the system stores a blob
+indefinitely.
+
+The system uploads a blob in *slivers*, which are small pieces of the file the system stores on
+different servers through *erasure coding*.
+[Learn more](https://docs.wal.app/design/encoding.html) about the Walrus architecture and how the
+system implements erasure coding.
+
+After you upload a blob to Walrus, it has 2 identifiers:
+
+```bash
+Blob ID: oehkoh0352bRGNPjuwcy0nye3OLKT649K62imdNAlXg
+Sui object ID: 0x1c086e216c4d35bf4c1ea493aea701260ffa5b0070622b17271e4495a030fe83
+```
+
+- Blob ID: A way to reference the blob on Walrus. The system generates the blob ID based on the
+  blob's contents, meaning any file you upload to the network twice results in the same blob ID.
+- Sui Object ID: The blob's corresponding newly created Sui object identifier, as the system binds
+  all blobs to one or more Sui objects.
+
+You use blob IDs to read blob data, while you use Sui object IDs to make modifications to the
+blob's metadata, such as its storage duration. You might also use them to read blob data.
+
+You can use the [Walrus Explorer](https://walruscan.com/) to view more information about a blob
+ID.
+
+## Retrieve a blob
+
+To retrieve a blob and save it on your local machine, run the following command:
+
+```bash
+walrus read <blob-id> --out file.txt --context testnet
+```
+
+Replace `<blob-id>` with the blob's identifier the `walrus store` command returns in its output,
+and replace `file.txt` with the name and file extension for storing the file locally.
+
+## Extend a blob's storage duration
+
+To extend a blob's storage duration, you must reference the Sui object ID and indicate how many
+epochs you want to extend the blob's storage for.
+
+Run the following command to extend a blob's storage duration by 3 epochs. You must use the Sui
+object ID, not the blob ID:
+
+```bash
+walrus extend --blob-obj-id <blob-object-id> --epochs-extended 3 --context testnet
+```
+
+Replace `<blob-object-id>` with the blob's Sui object ID the `walrus store` command returns in its
+output.
+
+## Delete a blob
+
+To delete a blob, run the following command:
+
+```bash
+walrus delete --blob-id <blob-id> --context testnet
+```
+
+Replace `<blob-id>` with the blob's identifier the `walrus store` command returns in its output.
+
+## Next steps
+
+1. [Build your first Walrus application](https://docs.wal.app/dev-guide/dev-guide.html). Explore
+   working examples:
+   - [Python examples](https://github.com/MystenLabs/walrus/tree/main/docs/examples/python)
+   - [JavaScript web form](https://github.com/MystenLabs/walrus/tree/main/docs/examples/javascript)
+   - [Move smart contracts](https://github.com/MystenLabs/walrus/tree/main/docs/examples/move)
+
+## Need help?
+
+- [Troubleshooting guide](https://docs.wal.app/usage/troubleshooting.html)
```

Comment:
Nit: We can reference this with an internal link:

```suggestion
- [Troubleshooting guide](./troubleshooting.html)
```

We could also add the following here or on the troubleshooting page:
```md
If you observe a bug or want to request a feature, please search for an existing
[issue](https://github.com/MystenLabs/walrus/issues) on this topic and, if none exists, create a new
one.
```